synapse/models/orgs.py

```diff
@@ -734,6 +756,51 @@ def getModelDefs(self):
                     ('org', ('ou:org', {}), {}),
                     ('name', ('ou:name', {}), {}),
                 )),
+
+                ('ou:asset:type:taxonomy', {}, ()),
+                ('ou:asset', {}, (
+                    ('org', ('ou:org', {}), {
+                        'doc': 'The organization which owns the asset.'}),
+
+                    ('id', ('str', {'strip': True}), {
+                        'doc': 'The ID of the asset.'}),
+
+                    ('name', ('str', {'strip': True, 'onespace': True}), {
```

Comment:
Don't need strip with onespace
```suggestion
                    ('name', ('str', {'onespace': True}), {
```

Reply:
Thanks, i keep forgetting that :)